Public/Src/Engine/UnitTests/Scheduler/PipFingerprinterTests.cs

```diff
@@ -92,7 +92,7 @@ public void ProcessStaticFingerprinting()
                 (fingerprinter, pip) => fingerprinter.ComputeWeakFingerprint(pip));
         }
 
-        [Fact]
+        //[Fact]

```

Comment:
>//[ [](start = 8, length = 3)

internional? If so, perhaps a:
`[Fact(Skip = ".....here's why...")]`
would be usefull

Reply:
ignore, should have read pr message first :)

---
In reply to: [301231301](https://github.com/microsoft/BuildXL/pull/552#discussion_r301231301) [](ancestors = 301231301)